packages/client/src/utils.ts

```diff
@@ -73,3 +76,49 @@ export function encodeCsvRow(items: any[]): string {
     })
     .join(",");
 }
+
+/**
+ * Given a observation series `num`, computes the ratio of the each
+ * observation value in `num` to the observation value in `denom` with
+ * the closest date value.
+ *
+ * Both `num` and `denom` series are sorted.
```

Comment:
Are these sorted by date (as opposed to value)? Please specify in this spec.

Reply:
done!